content/release/ReleaseNotes366.md

```diff
@@ -0,0 +1,63 @@
+<!--
+title: "Contrast 3.6.6 - July 2019"
+description: "Contrast 3.6.6 July 2019"
+tags: "3.6.6 July Release Notes"
+-->
+
+## Fixes
+
+* Application and libraries grids in the UI flickered for users on Safari. 
+* Some users had an issue onboarding multiple .NET applications in the same pool.
+* Impersonating users as a SuperAdmin resulted in errors.
+* Users without applications couldn't see custom access groups in the UI.
+* Applications tags were missing after applications were unmerged.
+* Application queries timed out in some organizations.
+* Daily digest emails were sent multiple times to the same recipients.
+* Filters weren't applied when using the "Select All" option to delete vulnerabilities or exporting vulnerabilities for merged applications.
+* Jira integrations didn't detect custom field to add multiple users.
+* Merged applications ignored an Assess rule disabled previously for one of the applications.
+
+## Improvements
+
+* Export your application’s route coverage information to use and share outside the UI. To download the spreadsheet, go to the application’s **Route Coverage** tab, and select the icon above the grid to **Export Routes to CSV**. You can also export the same data via the APIs.
+
+* The Node agent is available in the default Node Pivotal Cloud Foundry buildpack. You can also set up the Contrast tile to send any vulnerabilities up to your organization.
+
+* The Java agent now offers full support of IBM JDK 8 for Assess and Protect. 
```

Comment:
"full support" is a bit strong here. We verified that we support IBM JDK 8 at the same level that we support Open JDK 8.

Reply:
@rszewczyk Should I revise to just "supports IBM JDK 8", or take out the item altogether?